setup.py

```diff
@@ -60,6 +60,8 @@
           # sqlalchemy-redshift 0.7.7 introduced support for Parquet
           # in UNLOAD
           'sqlalchemy-redshift>=0.7.7',
+          # TODO https://github.com/jmcarp/sqlalchemy-postgres-copy/pull/14
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker